packages/core/src/app.module.ts

```diff
@@ -10,13 +10,12 @@ import {
   OperatorService,
   ChainService,
   EthModule,
-  EventModule,
```

Comment:
Ah I see so you removed EventModule because it's now in the eth-event service. Seems reasonable!